packages/runtime/src/next/tests/client.error-handling.test.ts

```diff
@@ -0,0 +1,57 @@
+import { HttpResponse } from 'msw'
+import { failedResponseBody } from '../client'
+
+describe('failedResponseBody', () => {
```

Comment:
Unit tests for the [`failedResponseBody` utility](https://github.com/makeswift/makeswift/pull/991/files?diff=split&w=1#diff-47636c8874de1aaa13ff42ed6ee56694d80e56961280ccd545dbc5cee42ff203R217).